h/accounts/__init__.py

```diff
@@ -1,4 +1,8 @@
 # -*- coding: utf-8 -*-
+from itsdangerous import URLSafeTimedSerializer
```

Comment:
This should just be `import itsdangerous`, see <https://google.github.io/styleguide/pyguide.html#Imports>